### PR TITLE
testing/py-scikit-learn: new aport

### PR DESCRIPTION
**Pull request by cgiraldo**

### CODE REVIEW

**Comment by Ikke** [comment]
Hello @cgiraldo,

You accidentally pushed a merge commit along with the actual commit.

Kind regards, Kevin

File: testing/py-scikit-learn/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	local python="$1"
+	pkgdesc="$pkgdesc ${python#python}"
+	depends="$depends $python"
+	install_if="$pkgname=$pkgver-r$pkgrel $python"
+
+	cd "$builddir"
+	$python setup.py install --prefix=/usr --root="$subpkgdir"
+}
+
+_py2() {
```
Comment (Ikke) [inline testing/py-scikit-learn/APKBUILD]: Is there a reason to add a python2 package? Python2 will be EOL soon, so we prefer new packages to only provide python3 variants unless python2 is required for something.

File: testing/py-scikit-learn/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Contributor: Carlos Giraldo <cgiraldo@gradiant.org>
+# Maintainer: Carlos Giraldo <cgiraldo@gradiant.org>
+pkgname=py-scikit-learn
+_pkgname=scikit-learn
+pkgver=0.20.3
+pkgrel=0
+pkgdesc="scikit-learn is a Python module for machine learning built on top of SciPy"
+url="http://scikit-learn.org/"
+arch="all"
+license="BSD"
```
Comment (Ikke) [inline testing/py-scikit-learn/APKBUILD]: The license should be a valid [SPDX identifier][SPDX], which just `BSD` is not. Please clarify which actual license it is.

[SPDX]:https://spdx.org/licenses/

File: testing/py-scikit-learn/APKBUILD (hunk 1 above)
Comment (cgiraldo) [reply testing/py-scikit-learn/APKBUILD]: I agree. 
I just dropped the python2 package.

File: testing/py-scikit-learn/APKBUILD (hunk 2 above)
Comment (cgiraldo) [reply testing/py-scikit-learn/APKBUILD]: Changed to BSD-3-Clause

**Pull request closed without merging**